dockerfiles/analytics/Dockerfile

```diff
@@ -16,8 +16,77 @@
 #
 # ------------------------------------------------------------------------
 
-# set to product base image
-FROM wso2ei-base:6.1.1
+# set to latest Ubuntu LTS
+FROM ubuntu:16.04
+MAINTAINER WSO2 Docker Maintainers "dev@wso2.org"
+
+# set user configurations
+ARG USER=wso2carbon
+ARG USER_ID=802
+ARG USER_GROUP=wso2
+ARG USER_GROUP_ID=802
+ARG USER_HOME=/home/${USER}
+# build arguments for WSO2 product installation
+ARG WSO2_SERVER_NAME=wso2ei
+ARG WSO2_SERVER_VERSION=6.1.1
+ARG WSO2_SERVER_REPOSITORY=product-ei
+ARG WSO2_SERVER=${WSO2_SERVER_NAME}-${WSO2_SERVER_VERSION}
+ARG WSO2_SERVER_HOME=${USER_HOME}/${WSO2_SERVER}
+ARG WSO2_SERVER_DIST_URL
+# set jdk configurations
+ARG JDK_NAME=jdk-8u251-linux-x64.tar.gz
+ARG JAVA_HOME=${USER_HOME}/java
+ARG JDK_URL
+# build arguments for external artifacts
+ARG MYSQL_CONNECTOR_VERSION=5.1.49
+# build argument for MOTD
+ARG MOTD="\n\
+Welcome to WSO2 Docker resources.\n\
+------------------------------------ \n\
+This Docker container comprises of a WSO2 product, running with its latest GA release \n\
+which is under the Apache License, Version 2.0. \n\
+Read more about Apache License, Version 2.0 here @ http://www.apache.org/licenses/LICENSE-2.0.\n"
+
```

Comment:
@RidmiR shall we keep only one extra line between each Dockerfile instruction. Please check other files as well.